documentation/library_guide/kernel_templates/esimd/radix_sort.rst

```diff
@@ -0,0 +1,338 @@
+| ``std::uint8_t RadixBits``  | The number of bits to sort per a radix sort algorithm pass.                           |
++-----------------------------+---------------------------------------------------------------------------------------+
+
+
+.. _parameters:
+
+Parameters
+----------
+
++-----------------------------------------------+---------------------------------------------------------------------+
+| Name                                          | Description                                                         |
++===============================================+=====================================================================+
+|  ``q``                                        | SYCL* queue to submit the kernels to.                               |
++-----------------------------------------------+---------------------------------------------------------------------+
+|                                               | The sequences of elements to apply the algorithm to.                |
+|  - ``r`` (1)                                  | Supported sequence types:                                           |
+|  - ``first``, ``last`` (2)                    |                                                                     |
+|  - ``keys``, ``values`` (3)                   | - ``sycl::buffer`` (1,3),                                           |
+|  - ``keys_first``, ``keys_last``,             | - ``oneapi::dpl::experimental::ranges::views::all`` (1,3),          |
+|    ``values_first`` (4)                       | - ``oneapi::dpl::experimental::ranges::views::subrange`` (1,3),     |
+|                                               | - USM pointers (2,4),                                               |
+|                                               | - ``oneapi::dpl::begin`` and ``oneapi::dpl::end`` (2,4).            |
+|                                               |                                                                     |
++-----------------------------------------------+---------------------------------------------------------------------+
+|  ``param``                                    | A :doc:`kernel_param <../kernel_configuration>` object.             |
+|                                               | Its ``data_per_workitem`` must be a positive multiple of 32.        |
+|                                               |                                                                     |
+|                                               |                                                                     |
++-----------------------------------------------+---------------------------------------------------------------------+
+
+
+**Type Requirements**:
+
+- The element type of sequence(s) to sort must be any
+  C++ integral and floating-point type with a width of up to 64 bits, except for ``bool``.
```

Comment:
I would change "must be any" to:
"may be any",

Alternatively, you could say something like:
"The element type of sequence(s) to sort must be a C++ integral and floating-point type other than ``bool`` with a width of up to 64 bits."

Reply:
I edited above to catch missed "and" -> "or" suggestion.